templates/release-semantic-automated-npm/release.config.ejs

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  "branches": <%- branches %>,
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    "@semantic-release/changelog",
+    ["@semantic-release/npm", {
+      "tarballDir": "release",
+      "npmPublish": false
+    }],
+    <% if (mainFilename) { %>
+     ["@semantic-release/exec", {
+        "prepareCmd": "sed -i \"s/Version:.*/Version:     ${nextRelease.version}/\" \"<%- mainFilename %>\" ",
+      }],
+    <% } %>
```

Comment:
`mainFilename` can only be `index.php` or `style.css`. So why add all this logic in the workflow to determine the correct file if we could simply run the search-replace command on both files?
```suggestion
    ["@semantic-release/exec", {
      "prepareCmd": "sed -i \"s/Version:.*/Version:     ${nextRelease.version}/\" index.php style.css 2> /dev/null",
    }],
```
The `2> /dev/null` is to silence the error that one of the files doesn't exist.
(untested!)

Reply:
Done, but `2> /dev/null` was outputting an exit code != 0. I had to tweak it a little to apply the idea